Extension/src/LanguageServer/extension.ts

```diff
@@ -1131,6 +1133,15 @@ function onRescanWorkspace(): void {
     clients.forEach(client => client.rescanFolder());
 }
 
+function onShowRefCommand(arg?: ReferenceItem): void {
+    if (arg) {
+        const { location } = arg;
+        vscode.window.showTextDocument(location.uri, {
+            selection: location.range.with({ end: location.range.start })
```

Comment:
We could send back the end or the source string to get the full range later.

Reply:
Having the full range of the symbol result would mean we will highlight in the editor. A selection of a confirmed reference already highlights the symbol. Should we highlight two different selections?